src/main/java/org/anastasiiapanchenko/lesson6/controller/TodoController.java

```diff
@@ -0,0 +1,51 @@
+package org.anastasiiapanchenko.lesson6.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.anastasiiapanchenko.lesson6.dto.TaskHistoryResponseDto;
+import org.anastasiiapanchenko.lesson6.dto.TodoCreateDto;
+import org.anastasiiapanchenko.lesson6.dto.TodoResponseDto;
+import org.anastasiiapanchenko.lesson6.dto.TodoUpdateDto;
+import org.anastasiiapanchenko.lesson6.entity.Todo;
+import org.anastasiiapanchenko.lesson6.service.TaskHistoryService;
+import org.anastasiiapanchenko.lesson6.service.TodoService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import jakarta.validation.Valid;
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/todos")
+public class TodoController {
+    private final TodoService todoService;
+    private final TaskHistoryService taskHistoryService;
+
+    @PostMapping
+    public ResponseEntity<TodoResponseDto> createTodo(@Valid @RequestBody TodoCreateDto todoCreateDto) {
+        TodoResponseDto createdTodo = todoService.createTodo(todoCreateDto);
+        return ResponseEntity.status(201).body(createdTodo); // Status 201 (Created)
```

Comment:
201 is magic number. Please use corresponding option: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/HttpStatus.html